tests/testthat/test-c-api.R

```diff
@@ -174,3 +174,55 @@ test_that("r_which_operator() returns correct tokens", {
   expect_identical(which_operator(quote(`(-`(a))), "")
   expect_identical(which_operator(quote(`{-`(a))), "")
 })
+
+test_that("client library passes tests", {
+  # Silence package building and embedded tests output
+  temp <- file()
+  sink(temp)
+  on.exit({
+    sink()
+    close(temp)
+  })
+
+  ## tools::testInstalledPackage() can't find the test package when
+  ## working with a temp lib:
+
+  ## old_libpaths <- .libPaths()
```

Comment:
@gaborcsardi The test globally works but if I uncomment this paragraph to install the embedded package in a temporary library, `tools::testInstalledPackage()` can't find the package.

Reply:
What happens if you pass the temp library directly to `lib.loc` rather than using `.libPaths()`?

Reply:
Same thing, it can't find the package.

Reply:
I would also not use `devtools` to do the install, just use `install.packages("pkg_path", repos = NULL, lib = temp_lib)`.

Reply:
probably also need `INSTALL_ops = "--install-tests"`

Reply:
Thanks! I didn't know `install.packages()` would do the building. Got rid of the devtools `Suggest`. It still can't find the package in the temporary lib though :/